aiidalab_widgets_base/wizard.py

```diff
@@ -0,0 +1,183 @@
+"""The wizard application allows the implication of a Wizard-like GUI.
+
+Authors:
+
+    * Carl Simon Adorf <simon.adorf@epfl.ch>
+"""
+from enum import Enum
+
+import traitlets
+import ipywidgets as ipw
+
+
+class WizardApp(ipw.VBox):
```

Comment:
```suggestion
class WizardAppWidget(ipw.VBox):
```

Since it is a widget, I would propose to rename it like this. Not all the existing widgets are renamed like this, but I would suggest we use the `...Widget` naming convention.

Reply:
Done.